src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadYamlStep.java

```diff
@@ -60,8 +61,10 @@
 public class ReadYamlStep extends AbstractFileOrTextStep {
 
 	public static final String DEFAULT_MAX_ALIASES_PROPERTY = ReadYamlStep.class.getName() + ".DEFAULT_MAX_ALIASES_FOR_COLLECTIONS";
+	@SuppressFBWarnings(value={"MS_SHOULD_BE_FINAL"}, justification="It's almost final. Can be set using a system property.")
```

Comment:
Improper justification? It could be final if it were just the system property, but it's probably nonfinal for Groovy console.

Reply:
not sure what you mean with the groovy console bit.

@rsandell - any reason they cannot or should not be final? I didn't question it because I was imagining some magic happening in jenkins core to set the value based upon a system property, but it just [Integer.getInteger(java.lang.String,int)](https://docs.oracle.com/javase/7/docs/api/java/lang/Integer.html#getInteger(java.lang.String,%20int)) :shrug:

Reply:
It is non final so that an admin can adjust the value through the [groovy script console](https://www.jenkins.io/doc/book/managing/script-console/) without restarting the instance.

Reply:
Ah, of course. Makes sense, thanks @StefanSpieker. Corrected in #166